matching-service/src/lib/hasher.ts

```diff
@@ -0,0 +1,9 @@
+import crypto from 'crypto';
+
+export const generateMatchId = (userId: string, matchId: string): string => {
```

Comment:
I think the indentation here is 4 spaces instead of 2, can run prettier to fix this

Reply:
Nice function tho

Reply:
fixed. One a sidenote, how are you guys seeing this linting issues? I cannot see them on my side.

Reply:
Have you set up husky?